src/update.h

```diff
@@ -117,7 +122,7 @@ typedef struct updateResult {
 
 /** structure describing an HTTP update job */
 typedef struct updateJob {
-	updateRequestPtr	request;
+	UpdateRequest		*request;
 	updateResultPtr		result;
```

Comment:
You have replaced `updateRequestPtr` in some places but not others. But maybe it is intentional ?